.github/workflows/stack-integration_tests.yml

```diff
@@ -85,6 +85,72 @@ jobs:
         run: |
           tox -e stack.test.integration
 
+  stack-integration-tests-public:
+    strategy:
+      max-parallel: 3
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+
+    runs-on: ${{matrix.os}}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Check for file changes
```

Comment:
Lets not trigger it based on file changes, lets always run it and move it to nightlies.